sig-index.md

```diff
@@ -11,3 +11,17 @@ Don't see a SIG here that fits what you're looking for? [Submit a Proposal](sig-
 * [Spinnaker-as-Code](sig-spinnaker-as-code/README.md)
 * [Summit](sig-summit/README.md)
 * [UI / UX](sig-ui-ux/README.md)
+
+# SIG Meeting Calendar
+
+To view the calendar, [click here](https://calendar.google.com/calendar/b/3/embed?showPrint=0&showCalendars=0&mode=AGENDA&height=500&wkst=1&bgcolor=%23FFFFFF&src=spinnaker.io_tr65rjf5mfij7p6vucprkhulcc%40group.calendar.google.com&color=%2342104A&ctz=America%2FLos_Angeles).
```

Comment:
When I click this link, the calendar is titled "Michael Plump"...is there some way to change that?  Also, I wonder if we want to remove the `ctz=` field from the URL so that it defaults to the user's timezone and not always America/Los_Angeles.

Reply:
I noticed the "Michael Plump" thing too. I pinged @plumpy about it in the sig-docs channel. 

I can remove the `ctz` part. It's just a carryover from what was on the .io page.